Content.Benchmarks/ColorInterpolateBenchmark.cs

```diff
@@ -27,7 +30,7 @@ public class ColorInterpolateBenchmark
         [GlobalSetup]
         public void Setup()
         {
-            var random = new Random(3005);
```

Comment:
You shouldn't use IoC random here:

1. for a benchmark, ideally the value is reproducible, so a set seed is good.
2. IoC isn't initialized here.

Reply:
Oh, right. I forgot to run those to check if they still worked